eiffel-vocabulary/EiffelIssueAssignedEvent.md

```diff
@@ -0,0 +1,242 @@
+<!---
+   Copyright 2017 Jaden Young
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# EiffelIssueAssignedEvent (IA)
+The EiffelIssueAssignedEvent declares that an issue defined by an
+[EiffelIssueDefinedEvent](./EiffelIssueDefinedEvent.md) has been assigned to a
+contributor. The fields of an __assignee__ in this event are intended to mirror
+the fields of an __author__ in
+[EiffelSourceChangeCreatedEvent](./EiffelSourceChangeCreatedEvent.md) to
+facilitate traceability.
+
+## Data Members
```

Comment:
Not necessarily a request for change, but a reflection. We are now creating the same problem for people, as we are solving for issues: multiple events referencing people with free-form strings. Is the next step an EiffelPersonIdentifiedEvent?